editor/plugins/script_editor_plugin.cpp

```diff
@@ -2856,6 +2856,8 @@ void ScriptEditor::_add_callback(Object *p_obj, const String &p_function, const
 
 		break;
 	}
+
+	EditorNode::get_singleton()->previous_item();
```

Comment:
This might deserve a comment, as it makes an assumption about editing history.

Reply:
Done!